### DISCUSSION TITLE
Internal Localization variables not found when using WixUI features

### DISCUSSION THREAD

**Discussion by MikeFarrington** [wixtoolset/issues#7692] [Questions]
This is similar to #6627, but not exactly.

I was experimenting with adding localization to my project and kept running across the error outlined in #6627 (but his solution did not apply to me).  

I am confronted with 551 errors about all of these localization variables that should be found in the UI extensions, but are not.

Curiously, the error messages indicate that it is looking for the files in a location on a developer's machine (or build server) as I do not have a "D:\a\wix\wix\src\ext\UI\wixlib" folder on my computer.  I have not downloaded or worked on any WiX Toolset sources.

![image](https://github.com/wixtoolset/issues/assets/40607634/df0ca700-a2ff-4ec7-a811-d9bd1acf774a)

To reproduce, start with a new project template as created in Visual Studio for an MSI package, adding the following code to the package (along with the associated nuget package and xmlns):
```
<ui:WixUI Id="WixUI_FeatureTree" />
```

I believe this is a bug worthy of an issue, but I didn't want to jump straight to that conclusion and create an issue in the main repo.

**Comment by robmen** [comment]
> To reproduce, start with a new project template as created in Visual Studio for an MSI package, adding the following code to the package (along with the associated nuget package and xmlns):
> 
> <ui:WixUI Id="WixUI_FeatureTree" />

I just tried this, and it worked for me.

**Reply by MikeFarrington** [reply]
I'm sorry Rob, I missed a reproduction step.  In the sample project, please copy the "Package.en-us.wxl" file to a new "Package.en-gb.wxl" file (and change the culture in that file to "en-GB").
